src/pspm_select_channels.m

```diff
@@ -40,7 +40,7 @@
     end
   case 'char'
     % in this case channel is specified as a char
-    if ~any(contains([{settings.channeltypes.type}, 'none', 'wave', 'events'], channel))
+    if ~any(startsWith([{settings.channeltypes.type}, 'none', 'wave', 'events'], channel))
```

Comment:
The update seems to allow some channels like `none_a`, which do not seem to be right? Would it be better to use `~any(startsWith([{settings.channeltypes.type}], channel)) || ~any(contains(['none', 'wave', 'events'], channel))`?

Reply:
| test channel type name | result |
|:--|:--|
| 'a' |  1 |
| 'none' | 0  |
| 'none_a' |  |
| "none" |  |
| 'scr' |  |
| 'scr_pp' |  |